README.rst

```diff
@@ -830,14 +830,15 @@ Password reset
 ==============
 
 If a user has registered an email address to their account using an identity
-server, they can request a password-reset token via clients such as Vector.
+server, they can request a password-reset token via clients such as Riot.im.
 
 A manual password reset can be done via direct database access as follows.
 
 First calculate the hash of the new password::
 
     $ source ~/.synapse/bin/activate
-    $ ./scripts/hash_password
+    $ cd ~/.synapse
+    $ ./bin/hash_password
```

Comment:
After using `source` above you should be able to just use `hash_password`

Reply:
The misunderstanding is as follows. I expected typing  `./script/hash_password` would call the script, but it doesn't. Apparently the script is stored in the bin directory, so after sourcing the command it can be called *without* any sub-directory prefix as `hash_password`. Not by `./scripts/hash_password` (as far as I understood the installation procedure correctly).

Reply:
hash_password is definitely in `scripts`, not `bin`: https://github.com/matrix-org/synapse/tree/master/scripts

Reply:
oh I see, you're using the version installed in the virtualenv. I guess that should work too.

`./script/hash_password` should work fine too though - what was the problem?